mods/ra/maps/soviet-04b/main.lua

```diff
@@ -0,0 +1,148 @@
+CheckForRadar = function()
+	Radar = Map.ActorsInBox(Map.TopLeft, Map.BottomRight, function(actor)
```

Comment:
This is criticized in https://github.com/OpenRA/OpenRA/issues/8549. In this case where the target is static, it is far easier to just reference the Actor `Radar` which automatically is a global in Lua.